### ISSUE TITLE
WG CR review for mediacapture-depth

### ISSUE THREAD

**Issue by alvestrand** [w3c/mediacapture-depth#133]
This issue serves to track internal WG review (in the WGs making up the mediacapture TF) of the <specification> document for CR publication.
If you have reviewed the document, please add a “thumbs up” reaction to the issue.
If you find issues that you think need addressing before CR publication, please file the issue in Github and mention it in the comments.

The review lasts until Thursday, October 12, 2016. At that time, the chairs will decide whether or not the review result warrants asking the wider community to review the document for CR; if we do ask the wider community, a similar issue wil be filed for tracking that review.

**Comment by fluffy** [comment]
I do not think this draft is ready for broad review and I have filed multiple issues

**Comment by anssiko** [comment]
Thanks for taking your time to review and report the issues. We'll address them and ping this meta issue when done. I'd suggest we go to wide review only after the issues reported during this review period have been addressed.
